Tweaks to allow test pass.
This version is dead. Will not send upstream.

--- a/testsuite/config/unix.exp	2017-05-15 11:00:07.860395459 +0000
+++ b/testsuite/config/unix.exp	2017-05-15 11:00:24.243034208 +0000
@@ -101,7 +101,7 @@
     #	return 0
     # }
     # Capture only stderr in exec_output, not "creating Makefile" etc.
-    catch "exec ./$args --cache=/dev/null >/dev/null" exec_output
+    catch "exec ./$args >/dev/null" exec_output
     if $verbose>1 then {
 	send_user "Executed $args --cache=/dev/null\n"
     }
